src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```

Comment:
constructor of `Animal` class does not take `health` as an argument according to task. It should be set to 100 by default.